src/sentry/api/endpoints/project_similar_issues_embeddings_records.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.tasks.backfill_seer_grouping_records import backfill_seer_grouping_records
+
+
+
+@region_silo_endpoint
+class ProjectSimilarIssuesEmbeddingsRecords(ProjectEndpoint):
```

Comment:
what do you think about adding `Backfill` somewhere in this endpoint name? (and in the url path in urls.py?) unless we want to add other functionality to this endpoint

Reply:
yeah that's a good idea - I'll make this change